README.md

```diff
@@ -79,68 +83,161 @@ following fixtures:
 
 import pytest
```

Comment:
Maybe put the conftest template in a dedicated file and refer to it in the README ?
Just a suggestion, I'm not even sure it would be better

Reply:
Done